package.json

```diff
@@ -5,7 +5,7 @@
     "repository": "https://github.com/GPII/gpii-pouchdb",
     "license": "BSD-3-Clause",
     "scripts": {
-        "pretest": "node node_modules/rimraf/bin.js reports/* coverage/*",
```

Comment:
I'm puzzled by this part of the change since I don't think we had problems with this script on any platform. I always saw these directories being removed reliably.

Reply:
My thinking was that we could better exercise the new approach by using it here, and also that we could avoid retaining rimraf as a dependency.  I could also simply make rimraf a dev dependency, as we already do nearly everywhere else.